flight/Libraries/alarms.c

```diff
@@ -238,6 +238,150 @@ static int32_t hasSeverity(SystemAlarmsAlarmOptions severity)
     return 0;
 }
 
+static const char alarm_names[][8] = {
```

Comment:
That'll use 4 bytes of flash per entry to save at most 7 bytes of flash per entry.  Not sure it's a win.  Of course, then the null would always be there.